docs/docs-content/clusters/edge/trusted-boot/trusted-boot.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: "Trusted Boot"
+title: "Trusted Boot"
+description: "Learn about Trusted Boot."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["edge"]
+---
+
+Trusted Boot is a security feature supported by Palette Edge running on Edge devices with supported hardware and
+firmware. Trusted Boot consists of the following security measures:
+
+- Full disk encryption (FDE): Encryption of the entire contents of a disk drive, including the operating system, system
+  files, and user data. The purpose of FDE is to protect data stored on the disk from unauthorized access in the event
+  of theft or loss of the device.
+- Secure boot: A security measure that ensures only properly signed and authenticated software is allowed to operate
+  during the boot process of a device.
+- Measured boot. A security feature that works by measures each component of the boot process and recording these
+  measurements in a
+  [Trusted Platform Module (TPM)](https://www.intel.com/content/www/us/en/business/enterprise-computers/resources/trusted-platform-module.html),
+  which can be used for later analysis.
+
+Together, these measures allows Trusted Boot to ensure the authenticity of the boot processes that are allowed to run on
+your Edge device. In addition, the measurements stored in the TPM can be provided to third parties to prove that your
+boot process is secure and unaltered. In the event that your Edge device is stolen or lost, FDE ensures that its content
```

Comment:
actually, in the event the device is stolen or lost, trusted boot + encryption ensures that it is very difficult to steal the data. This is because the thief can't alter the boot process (which also determines who the users etc. are), meaning they can't inject e.g. a new root user and then decrypt the drive. that's because the TPM won't release the key for the encrypted portion if the UKI is tampered with, and the key is only stored inside the TPM.
The end result is kind of similar to what you wrote, but it's the combination of encryption and TB that prevents the theft of data.